geth/jail/handlers.go

```diff
@@ -66,23 +66,54 @@ func registerHandlers(jail *Jail, cell common.JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail, cellInt common.JailCell) func(call otto.FunctionCall) otto.Value {
+	// FIXME(tiabc): Get rid of this.
```

Comment:
Hm? Can we get rid of this in this PR? I guess we can just use `cell *Cell`?

Reply:
Yes but it better be done during some refactoring as it would introduce a number of unrelated changes. Generally, I'd like to revisit how we use jail cell and its interface.